docs/specifications/methods/undelegated-test.md

```diff
@@ -0,0 +1,44 @@
+# Undelegated tests
+
```

Comment:
## Undelegate00: Routines to be performed when testing an undelegated domain

Reply:
The motivation for having a separate directory is that the specifications under "test" are connected to the test cases, whereas this specification defines how the environment is set for undelegated tests.